### PR TITLE
RIP Custom Cursors

### LINKED ISSUE #1147: Cursor color changes are distracting and broken
OS LInux
Browser Chrome 56 (64-bit)

The cursor color changes give it weird backgrounds and is quite distracting.

test

### PR DESCRIPTION
**Pull request by alexpaxton**
Fix #1147 

### The problem
Older version of chrome do not support PNGs with alpha transparency, cursors become very fugly and distracting

### The Solution
Switched to `.cur` format, which is supported at least as far back as Chrome 38. I dare not venture further.
Gecko browsers have supported PNG and CUR for a long time.

### UPDATE
Removing this until I can find a cleaner solution

### CODE REVIEW

**Comment by alexpaxton** [comment]
@nathanielc can you run this branch in your browser and see if the if the issue persists? I tested in a ton of old browsers and it looks fine, just want to double check

**Comment by lukevmorris** [comment]
TIL about the `.cur` file format.

File: ui/webpack/ignoreForTestLoader.js [hunk 1]
```diff
@@ -1,24 +0,0 @@
...
-
-// Tells webpack not to bother with other loaders in this chain.
-// See https://github.com/webpack/null-loader/blob/master/index.js
-module.exports.pitch = function() {
-  if (this.cacheable) {
-    this.cacheable();
-  }
-
-  return moduleSource;
-};
```
Comment (lukevmorris) [inline ui/webpack/ignoreForTestLoader.js]: What was this doing?

File: ui/webpack/testConfig.js [hunk 2]
```diff
@@ -1,60 +0,0 @@
...
-      app: path.resolve(__dirname, '..', 'app'),
-      src: path.resolve(__dirname, '..', 'src'),
-      chronograf: path.resolve(__dirname, '..', 'src', 'chronograf'),
-      shared: path.resolve(__dirname, '..', 'src', 'shared'),
-      style: path.resolve(__dirname, '..', 'src', 'style'),
-      utils: path.resolve(__dirname, '..', 'src', 'utils'),
-      sinon: 'sinon/pkg/sinon',
-    }
-  }
-};
```
Comment (lukevmorris) [inline ui/webpack/testConfig.js]: And dis

**Comment by lukevmorris** [comment]
couple questions above. If @nathanielc can verify his cursor looks ok, then it should be good to go

**Comment by nathanielc** [comment]
@alexpaxton Still not fixed on my system. I have confirmed that the .cur based images are being used but the behavior is still the same. :(

File: ui/webpack/ignoreForTestLoader.js (hunk 1 above)
Comment (alexpaxton) [inline ui/webpack/ignoreForTestLoader.js]: @cryptoquick determined these to be doing nothing, vestigial code

**Comment by alexpaxton** [comment]
@nhaugo removing these for now, it was a fun (and educational) trip down the rabbit hole

**Comment by lukevmorris** [comment]
Build should be green again

**Comment by alexpaxton** [comment]
@lukevmorris good catch, muchas gracias

**Pull request merged**